backend/fms_core/example_data/csv/samples_transfers.csv

```diff
@@ -0,0 +1,10 @@
+Sample Transfer Template,,,,,,,,,,,,
+,,,,,,,,,,,,
+,,,,,,,,,,,,
+,,,,,,,,,,,,
+,,,,,,,,,,,,
+,,,,,,,,,,,,
+,,,,,,,,,,,,
+#,Source Container Barcode,Source Location Coord,Destination Container Barcode,Destination Location Coord,Destination Container Name,Destination Container Kind,Destination Parent Container Barcode,Destination Parent Container Coord,Source Depleted,Volume Used (uL),Transfer Date,Comment
+1,tube001,,newtubefortransfer,,newtubefortransfer,Tube,,,,10,2021-01-01,Test Comment

```

Comment:
I would test cases with containers with coords and localize the destination container... It may help us catch errors down the line.